Library/.rubocop.yml

```diff
@@ -28,7 +28,7 @@ FormulaAuditStrict/Options:
   Enabled: true
 
 NewFormulaAudit/Options:
-  Enabled: false
+  Enabled: true 
```

Comment:
@GauthamGoli Worst case we can run the new formulae cops in a separate `rubocop` invocation.